### PR TITLE
LPS-194674 Update layout calculation based on new url format

### PR DESCRIPTION
**Pull request by victorg1991**
There are still things missing like testing this with proxy objects

### CODE REVIEW

**Comment by victorg1991** [comment]
ci:test:sf

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by victorg1991** [comment]
ci:test:relevant

**Comment by victorg1991** [comment]
Checking test failures, I believe those are related

**Comment by victorg1991** [comment]
ci:test:relevant

File: modules/apps/layout/layout-page-template-service/src/main/java/com/liferay/layout/page/template/internal/info/item/provider/DisplayPageInfoItemFieldSetProviderImpl.java [hunk 1]
```diff
@@ -108,6 +113,25 @@ public List<InfoFieldValue<Object>> getInfoFieldValues(
...
 
+			Map<Locale, String> urls = new HashMap<>();
+
+			Layout layout = _layoutLocalService.fetchLayout(
+				layoutPageTemplateEntry.getPlid());
+
+			Map<Locale, String> friendlyURLMap = layout.getFriendlyURLMap();
+
+			for (Map.Entry<Locale, String> entry : friendlyURLMap.entrySet()) {
+				Locale locale = entry.getKey();
```
Comment (ealonso) [inline modules/apps/layout/layout-page-template-service/src/main/java/com/liferay/layout/page/template/internal/info/item/provider/DisplayPageInfoItemFieldSetProviderImpl.java:124]: @victorg1991 you can inline it

File: modules/apps/asset/asset-display-page-api/src/main/java/com/liferay/asset/display/page/portlet/BaseAssetDisplayPageFriendlyURLResolver.java [hunk 2]
```diff
@@ -205,13 +205,8 @@ public LayoutFriendlyURLComposite getLayoutFriendlyURLComposite(
 		return new LayoutFriendlyURLComposite(layout, friendlyURL, false);
 	}
 
-	protected boolean useOriginalFriendlyURL() {
```
Comment (ealonso) [inline modules/apps/asset/asset-display-page-api/src/main/java/com/liferay/asset/display/page/portlet/BaseAssetDisplayPageFriendlyURLResolver.java]: @victorg1991 it should be sf of previous commits

**Comment by ealonso** [comment]
see: https://github.com/liferay-echo/liferay-portal/pull/13547

**Pull request closed without merging**